### DISCUSSION TITLE
Raster Particle Layer Support

### DISCUSSION THREAD

**Discussion by kevinmanncito** [rnmapbox/maps#3582] [General]
Hi! Any plans for implementing the `RasterParticleLayer` for handling animated raster layers? Looks like the native mapbox sdk has support for this (might only be in v11 though): 
https://docs.mapbox.com/ios/maps/api/11.5.1/documentation/mapboxmaps/rasterparticlelayer/

**Comment by mfazekas** [comment]
@kevinmanncito Nope it's not yet supported, PR-s welcome!

**Reply by kevinmanncito** [reply]
Fair enough! I'll see what I can do

**Reply by mihalcind** [reply]
Hey @kevinmanncito, do you have any progress with the implementation?

**Reply by kevinmanncito** [reply]
Unfortunately not yet, I need to get our app running on mapbox 11 first... Also creating your own raster layers with mapbox's MTS service requires them to give you access to their private beta which I'm also working on.

**Reply by mihalcind** [reply]
Maybe it'll be possible to use Mapbox example wind data [[android](https://docs.mapbox.com/android/maps/examples/raster-particles/), [ios](https://docs.mapbox.com/ios/maps/examples/swift-ui-raster-particle/)] for the testing.

**Reply by sethjhs9** [reply]
This feature would be amazing
@kevinmanncito what help do you need? @mfazekas Do you have a template PR for adding new source and layer classes? How do we handle the switch between features only in mapbox 11?
